src/services/routes/org.routes.ts

```diff
@@ -1,16 +1,28 @@
 import type { RouteOptions } from 'fastify';
-import isValidDomain from 'is-valid-domain';
 
 import { HTTP_STATUS_CODES } from '../http.js';
 import type { PluginDone } from '../types/PluginDone.js';
 import { ORG_SCHEMA } from '../schema/org.schema.js';
 import type { FastifyTypedInstance } from '../fastify.js';
-
-export enum ProblemType {
-  MALFORMED_ORG_NAME = 'https://veraid.net/problems/malformed-org-name',
-  MALFORMED_AWALA_ENDPOINT = 'https://veraid.net/problems/malformed-awala-endpoint',
+import { createOrg } from '../../org.js';
+import { CreationProblemType } from '../../CreationProblemType.js';
+
+const ORG_ROUTES_ERROR_MAPPING: {
+  [key in CreationProblemType]: (typeof HTTP_STATUS_CODES)[keyof typeof HTTP_STATUS_CODES];
```

Comment:
I'm sure you tried this, but just to double check: isn't there a simpler alternative to `(typeof HTTP_STATUS_CODES)[keyof typeof HTTP_STATUS_CODES]`?

Doesn't `keyof typeof HTTP_STATUS_CODES` work?

Reply:
Yep I tried that.
 keyof typeof HTTP_STATUS_CODES returns the keys of the object like "OK", "NO_CONTENT", etc...
 In this case we need the values not the keys that's why I had to write this

Reply:
Cool, let's leave it as it is then 👍🏾